keras/engine/training.py

```diff
@@ -1080,7 +1080,7 @@ def metrics(self):
             y, y_pred, sample_weight, regularization_losses=self.losses
         )
 
-    def compute_metrics(self, x, y, y_pred, sample_weight):
+    def compute_metrics(self, x, y, y_pred, sample_weight, training=False):
```

Comment:
Is there a reason why `training` defaults to `False`? If this is primarily meant to be called by the training library, having no default value and forcing the caller to be explicit seems preferred.

Reply:
Removed the default value here and elsewhere.